code/__defines/MC.dm

```diff
@@ -32,6 +32,7 @@ if (Datum.is_processing) {\
 
 #define STOP_PROCESSING(Processor, Datum) \
 if(Datum.is_processing) {\
+	Processor.stopProcessingWrapper(Datum);\
```

Comment:
Add in a comment here too, explaining that this wrapper exists to update the pivot when things stop processing

Reply:
Comment above the macro
This is the main entrypoint through which people will find this. a comment here is important